### DISCUSSION TITLE
I want to add klaytn Network.

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#187] [Q&A]
I want to use klaytn Network.
so i try to that.
But i see worng network.
what did i do wrong?

 var klayChain = new Chain(
                    "klaytn",
                    "8217",
                    "Klaytn",
                    new Currency("klay", "KLAY", 18),
                    new BlockExplorer("Klaytn Scope", "https://scope.klaytn.com"),
                    "https://public-en-cypress.klaytn.net",
                    false,
                    "https://assets.coingecko.com/coins/images/4380/standard/download.png"
                );

                var itemklay = Instantiate(_networkListItemPrefab, _networkListContainer);
                itemklay.Initialize(klayChain, OnNetworkSelected);

![image](https://github.com/WalletConnect/WalletConnectUnity/assets/109068040/94ce48ed-da06-417d-a570-522fe438fdb2)

**Comment by skibitsky** [comment]
Klaytn's network namespace is `eip155`. You can find a list of all chain IDs (in `chain namespace:chain reference` format) [here](https://docs.walletconnect.com/advanced/multichain/chain-list).

**Reply by matthew2099** [reply]
Thank you. i try that.

**Reply by matthew2099** [reply]
@skibitsky Thank you. I success that ㅎ
